squarelet_auth/migrations/0001_initial.py

```diff
@@ -11,6 +11,7 @@ class Migration(migrations.Migration):
     initial = True
 
     dependencies = [
+        ('accounts', '0005_remove_squareletuser_organizations_and_more'),
```

Comment:
i had to add this so the agenda watch migrations would run. i tried removing it and it worked briefly, but would break my migrations unpredictably. so now this repo is hyper-coupled to the agenda watch repo.

maybe a good technical debt payoff would be rewriting all of this repo within the agenda watch repo.